tests/qemu/task.yaml

```diff
@@ -6,8 +6,11 @@ prepare: |
         exit 1
     fi
 
+    apt install -y autopkgtest genisoimage
+    autopkgtest-buildvm-ubuntu-cloud -a amd64 -r xenial
+
     mkdir -p ~/.spread/qemu
-    ln -sf /dev/sda ~/.spread/qemu/ubuntu-16.04.img
+    mv autopkgtest-xenial-amd64.img ~/.spread/qemu/ubuntu-16.04.img
```

Comment:
What we used to have here would use a cached image. This is building the image every single time we run this test. If we have to debug this, we'll be paying the price of every single time too.

Reply:
Ok, I was thinking on run this just on google, but we really need also to be able to run that on qemu, I'll add the logic to keep both backends working.
Thanks for reviewing this